### DISCUSSION TITLE
Unable to render rich text

### DISCUSSION THREAD

**Discussion by joshiayush** [github-linguist/linguist#6244] [Classification] [answered]
Look at this file https://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/classification.ipynb, Why GitHub is rendering rich text for tensorflow and not for me like this one https://github.com/joshiayush/ai/blob/master/ai/algos/correlation/pearson_correlation/Pearson_Correlation_Coefficient.ipynb. Now, after spending two years with GitHub do I have to find an alternative of it where I can transfer all my projects and say fucking good bye to GitHub?

Neither it renders rich text for me nor it lists code files under the language section in my repo, see https://github.com/joshiayush/ai/search?l=jupyter-notebook. Now, I'm getting mad on GitHub because since I created that repo it is consistent with its fucking behaviour.

**Comment by lildude** [comment, marked as answer]
> Look at this file https://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/classification.ipynb, Why GitHub is rendering rich text for tensorflow and not for me like this one https://github.com/joshiayush/ai/blob/master/ai/algos/correlation/pearson_correlation/Pearson_Correlation_Coefficient.ipynb.

Linguist has no say in the rendering of Jupyter notebooks. It is only responsible for the language detection. If you want to know why your file isn't rendering, please contact GitHub support using the "Contact GitHub" link at the bottom of the page when viewing the file.

> Neither it renders rich text for me nor it lists code files under the language section in my repo, see https://github.com/joshiayush/ai/search?l=jupyter-notebook.

Linguist isn't responsible for this either. This is GitHub's search which is completely independent of Linguist (it uses [go-enry](https://github.com/go-enry/go-enry) for its language support which feeds off Linguist).
